src/components/chip/react/Chip.tsx

```diff
@@ -155,52 +152,51 @@ const Chip: React.FC<IChipProps> = ({
     };
 
     return (
-        <div
+        <a
             className={classNames(
                 className,
                 handleBasicClasses({
-                    active: Boolean(isActive),
                     clickable: Boolean(hasOnClick),
                     disabled: Boolean(isDisabled),
                     hasAfter: Boolean(hasAfterClick),
                     hasBefore: Boolean(hasBeforeClick),
                     prefix: CLASSNAME,
+                    selected: Boolean(isSelected),
                     size,
                     theme,
-                    unselected: Boolean(!isActive),
+                    unselected: Boolean(!isSelected),
                 }),
             )}
             role="button"
             tabIndex={isDisabled || !hasOnClick ? -1 : 0}
             onClick={onClick || noop}
-            onKeyPress={onClick}
+            onKeyDown={onEnterPressed(onClick || noop)}
```

Comment:
Maybe something like `onClick ? onEnterPress(onClick) : null` I think when a event prop is null it doesn't even attach the event. With noop it would still attach one but do nothing. What do you think?

Reply:
Right :+1: